### DISCUSSION TITLE
Some questions about modified version QE (5.1.0)

### DISCUSSION THREAD

**Discussion by ZhongyuWAN** [SSCHAcode/python-sscha#165] [Q&A] [answered]
Hello, developers.

I meet some problems with modified version QE (5.1.0).
I performed the SCF calculation with k = (8, 8, 8) to the system of interest and it works fine. However, when I use the program ph.x to generate the dynamic matrix and electron-phonon coupling matrix elements for q = (4, 4, 4), it interrupts at the 7-th q point. 

user/qe-5.1.0_elph/bin/ph.x < ph.in > ph.out

 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
     Error in routine set_irr_sym_new (422):
     wrong representation
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

     stopping ...

But I used the latest standard QE (7.0) to run the same input file.
user/qe7.0/bin/ph.x < ph.in > ph.out
The program runs normally and outputs all the q-points.

The second question is about the parameters of elph_fc.x. In the 8th tutorials, why do "minbroad" and "nbroad" take value 10cm-1 and 1. For other systems, how to test the convergence of minbroad and broad?

Thanks

**Comment by DjordjeDangic** [comment, marked as answer]
Hello Zhongyu,

This QE error usually happens due to the fact that quantum espresso is not always consistent when assigning symmetries to the same structure. It does not have anything to do with the modification made to the code. You could try changing slightly your input structures (giving more significant figures) and maybe in that case your problem would go away. Another way of doing it is to modify qe7.0 to print out the deformation potentials just as it is done for version 5.1.

You converge broadening as any other. Reduce it until you reach the plateau, if you go too small the result will stop making sense. nbroad if I remember correctly just tells code how many broadenings to check in steps of minbroad.
